stratum0.yml

```diff
@@ -0,0 +1,29 @@
+- name: CVMFS
+  hosts: cvmfsstratum0servers
+  vars:
+    cvmfs_numfiles: 4096
+    cvmfs_server_urls:
+      - domain: eessi-hpc.org
+        urls:
+          - "http://cvmfs-s0.eessi-hpc.org/cvmfs/@fqrn@"
+    cvmfs_repositories:
+      - repository: software.eessi-hpc.org
+        stratum0: cvmfs-s0.eessi-hpc.org
+        key_dir: /etc/cvmfs/keys/eessi-hpc.org.org
+        server_options:
+          - CVMFS_AUTO_TAG=false
+          - CVMFS_GARBAGE_COLLECTION=true
+          - CVMFS_AUTO_GC=false
+        client_options:
+          - CVMFS_NFILES=4096
+      - repository: cvmfs-config.eessi-hpc.org
+        stratum0: cvmfs-s0.eessi-hpc.org
+        key_dir: /etc/cvmfs/keys/eessi-hpc.org
+        server_options:
+          - CVMFS_AUTO_TAG=false
+          - CVMFS_GARBAGE_COLLECTION=true
+          - CVMFS_AUTO_GC=false
+        client_options:
+          - CVMFS_NFILES=4096
+  roles:
+    - galaxyproject.cvmfs
```

Comment:
Can we avoid using `galaxy`?

Reply:
Renamed the role (the submodule directory) from galaxyproject.cvmfs to just cvmfs, so that's what I use inside the playbooks now as well.